assets/less/content/general.less

```diff
@@ -73,18 +73,58 @@ a.view-source {
 
 blockquote {
   font-style: italic;
-  margin: .5em 0;
-  padding: .25em 1.5em;
   border-left: 3px solid @gray;
-  display: inline-block;
+  position: relative;
+  margin: 1.5625em 0;
+  padding: 0 1.2rem;
+  font-size: 1.28rem;
+  overflow: auto;
+  background-color: #f6f6f6;
+  border-radius: 3px;
 
   *:first-child {
```

Comment:
* [ ] TODO: Limit this behavior to `h2` and `h3` instead of every first-child in a `blockquote`

Reply:
I would go exclusively with h3,h4,..., which should also solve the issue with section headers, right?

Reply:
Done